src/components/Login/index.tsx

```diff
@@ -71,51 +74,52 @@ export const FormComp = (props: StyleProps & FormikProps<FormValues>) => {
   )
 }
 
-interface StyleProps {
-  classes: {
-    grid: string
-    form: string
-  }
-}
-
-const styles = createStyles({
-  form: {
-    display: "flex",
-    minHeight: "100vh",
-  },
-  grid: {
-    display: "flex",
-    justifyContent: "center",
-    alignContent: "center",
-    color: "red",
-  },
-})
-
-export const LoginForm = withStyles(styles)(FormComp)
+//export const LoginForm = withStyles(styles)(FormComp)
 
 //TODO: Research purpose of this interface
-interface MyFormProps {
-  initlogin?: ""
+interface MyFormProps extends RouteComponentProps {
```

Comment:
@Alexandria Did you ever research to figure out if this interface was needed or not?

Reply:
Yes, that interface is really only needed if I want to initialize my form values with some other input.  I don't need it right now so I will just remove it.